consensus/view_change.go

```diff
@@ -196,22 +196,24 @@ func (consensus *Consensus) onViewChange(msg *msg_pb.Message) {
 			msg_pb.MessageType_PREPARED, recvMsg.BlockNum,
 		)
 		preparedMsg := consensus.FBFTLog.FindMessageByMaxViewID(preparedMsgs)
-		block := consensus.FBFTLog.GetBlockByHash(preparedMsg.BlockHash)
```

Comment:
Damn. I remember I fixed this nil pointer case on Saturday. Somehow my fix is gone... or I missed it in this part of the code.

Reply:
and another, https://github.com/harmony-one/harmony/pull/2970/commits/5a37b785824c5ecc8b746a966446a39508328fbb, via 

11273]: node.sh: node process finished with status 2
11273]: /home/ec2-user/go/src/github.com/harmony-one/harmony/consensus/consensus_v2.go:336 +0x67
11273]: created by github.com/harmony-one/harmony/consensus.(*Consensus).Start
11273]: /home/ec2-user/go/src/github.com/harmony-one/harmony/consensus/consensus_v2.go:497 +0x652
11273]: github.com/harmony-one/harmony/consensus.(*Consensus).Start.func1(0xc0003dac80, 0xc0005acde0, 0xc0008162a0, 0xc0005acba0, 0xc0
11273]: /home/ec2-user/go/src/github.com/harmony-one/harmony/consensus/consensus_v2.go:91 +0x254
11273]: github.com/harmony-one/harmony/consensus.(*Consensus).handleMessageUpdate(0xc0003dac80, 0xc0095e2006, 0x199, 0x19a)
11273]: /home/ec2-user/go/src/github.com/harmony-one/harmony/consensus/view_change.go:263 +0xb0d
11273]: github.com/harmony-one/harmony/consensus.(*Consensus).onViewChange(0xc0003dac80, 0xc007166280)
11273]: /home/ec2-user/go/src/github.com/harmony-one/harmony/node/node_handler.go:291 +0x77
11273]: github.com/harmony-one/harmony/node.(*Node).VerifyNewBlock(0xc0001fb200, 0xc00ac64680, 0x1, 0xc00ac64680)
11273]: /home/ec2-user/go/src/github.com/harmony-one/harmony/core/block_validator.go:128 +0x4d
11273]: github.com/harmony-one/harmony/core.(*BlockValidator).ValidateHeader(0xc0001a4460, 0xc00ac64680, 0xc0001a4401, 0x200000003, 0x
11273]: /home/ec2-user/go/src/github.com/harmony-one/harmony/core/types/block.go:520
11273]: github.com/harmony-one/harmony/core/types.(*Block).Header(...)
11273]: /home/ec2-user/go/src/github.com/harmony-one/harmony/core/types/block.go:387
11273]: github.com/harmony-one/harmony/core/types.CopyHeader(...)


stack inverted because journalctl -r